app/database/accessor/breeze_buddy/blacklisted_numbers.py

```
@@ -0,0 +1,183 @@
+        try:
+            redis = await get_redis_service()
+            await redis.setex(
+                key=cache_key,
+                value="1" if is_blocked else "0",
+                ttl_seconds=BLACKLIST_CACHE_TTL,
+            )
+        except Exception as e:
+            logger.warning(f"Redis cache set error for blacklist: {e}")
+
+        return is_blocked
+    except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)